### PR TITLE
Added new userVerificationBioOrPin key

### PR DESCRIPTION
**Pull request by pavelkozlov-okta**
### Problem Analysis (Technical)

Add support of new `userVerificationBioOrPin` key.

### Solution (Technical)

Added a capability to register the new key. Push Factor is out of scope at this moment.

### CODE REVIEW

File: Sources/DeviceAuthenticator/Transactions/BiometricSettings.swift [hunk 1]
```diff
@@ -31,4 +31,9 @@ public struct BiometricEnrollmentSettings {
         let settings = BiometricEnrollmentSettings(accessControlFlags: accessControl)
         return settings
     }
+
+    // - Description: Constructs a BiometricEnrollmentSettings instance with enabled bio or passcode settings
+    public static var userPresence: BiometricEnrollmentSettings {
```
Comment (IldarAbdullin-okta) [inline Sources/DeviceAuthenticator/Transactions/BiometricSettings.swift:36]: Please remove. Should not be exposed in public sdk

File: Sources/DeviceAuthenticator/Enrollment/EnrollmentParameters.swift [hunk 2]
```diff
@@ -29,6 +29,15 @@ public struct EnrollmentParameters {
         self.userVerificationSettings = userVerificationSettings
     }
 
+    /// Enables/disables user verification with biometrics or pin capabilities for the enrollment
+    /// - Parameters:
+    ///   - enable: Boolean for enable/disable user verification capabilties
+    ///   - userVerificationBioOrPinSettings: User verification using biometry or pin settings for the enrollment
+    public mutating func enableUserVerificationBioOrPin(enable: Bool, userVerificationBioOrPinSettings: BiometricEnrollmentSettings = .userPresence) {
```
Comment (IldarAbdullin-okta) [inline Sources/DeviceAuthenticator/Enrollment/EnrollmentParameters.swift]: Please remove from public surface

**Review by IldarAbdullin-okta — COMMENTED** [review]
Overall looks good. Good tests!
Please address comments

**Review by moisesolmedo-okta — APPROVED** [review]

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM!

**Pull request merged**